x/ccv/provider/keeper/relay.go

```diff
@@ -162,6 +162,13 @@ func (k Keeper) EndBlockVSU(ctx sdk.Context) {
 	}
 }
 
+// BlocksUntilNextEpoch returns the number of blocks until the next epoch starts
+// Returns 0 if the current block is the last block of the epoch
```

Comment:
To not make the UX confusing, this should return 0 if the current block is the first block of the epoch.

Reply:
Good point, thanks. The comment was wrong, anyways. I adjusted both the numbering and the comment